odpf/assets/v1beta2/ml_feature.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package odpf.assets.v1beta2;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/assets/v1beta2;assetsv1beta2";
+option java_outer_classname = "MLFeatureProto";
+option java_package = "io.odpf.assets";
+
+// MLFeature is a Machine Learning(ML) feature. In machine learning, a feature
+// is an individual measurable property, typically represented by a column,
+// that serves as an input for Machine Learning (ML) algorithms.
+message MLFeature {
```

Comment:
@sudo-suhas lets call it just feature.

Reply:
I feel 'feature' would not be clear enough as it is an overloaded word.